onnx/defs/data_type_utils.cc

```diff
@@ -109,10 +109,13 @@ DataType DataTypeUtils::ToType(const std::string& type_str) {
 }
 
 const TypeProto& DataTypeUtils::ToTypeProto(const DataType& data_type) {
+  // If the DataType input is invalid, this function will return an empty TypeProto
```

Comment:
Could you add this in the class header data_type_utils.h where this function is declared.

Reply:
Updated!